kuma/wiki/templatetags/jinja_helpers.py

```diff
@@ -207,7 +207,7 @@ def document_zone_management_links(user, document):
 
     # Enable "add" link if there is no zone for this document, or if there's a
     # zone but the document is not itself the root (ie. to add sub-zones).
-    if ((not zone or zone.document != document) and
+    if ((not zone or zone.document_id != document.id) and
```

Comment:
@safwanrahman Again, given your change to the `DocumentNearestZoneJob.fetch` above, which populates the zone's related document, it seems this change is no longer helpful to save a query?

Reply:
Yes. it seems to be not saving a query anymore. but I thought abou deferred object can give wrong result in comparing, but it seems that they may make another query if value is not present.
So I think comparing by id would be feasible.

Reply:
@safwanrahman Yes, thanks.